dts/riscv/gigadevice/gd32vf103.dtsi

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2021 Tokita, Hiroshi <tokita.hiroshi@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <dt-bindings/timer/nuclei-machine-timer.h>
+
+/ {
+	#address-cells = <1>;
+	#size-cells = <1>;
+
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu: cpu@0 {
+			clock-frequency = <108000000>;
+			mcause-exception-mask = <0x7ff>;
+			compatible = "nuclei,bumblebee", "riscv";
+			device_type = "cpu";
+			reg = <0>;
+		};
+	};
+
+	sram0: memory@20000000 {
+		compatible = "mmio-sram";
+	};
+
+	soc {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		compatible = "gd,gd32vf103-soc", "simple-bus";
+		ranges;
+
+		mtimer: machine-timer@d1000000 {
+			compatible = "nuclei,machine-timer";
+			reg = <0xd1000000 0x1
+			       0xd1000008 0x1>;
+			clk-divider = <NUCLEI_MACHINE_TIMER_DIVIDER_4>;
```

Comment:
question: is 4 the SoC default?

Reply:
Yes. It is a default and also a fixed value. (It comes from hardware design. Details in http://www.gd32mcu.com/data/documents/shujushouce/GD32VF103_User_Manual_EN_V1.2.pdf#%5B%7B%22num%22%3A145%2C%22gen%22%3A0%7D%2C%7B%22name%22%3A%22XYZ%22%7D%2C60%2C769%2C0%5D 
)

In actuality, No need for definitions except for NUCLEI_MACHINE_TIMER_DIVIDER_4 at this time.

(Since I made it able to use various division ratios, I defined some constants that may be usable, but I think it is unlikely that it will use as far as gd32v is concerned .)